src/NetAnalyzers/CSharp/Microsoft.CodeQuality.Analyzers/Maintainability/CSharpUseCrossPlatformIntrinsics.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeQuality.Analyzers.Maintainability;
+
+namespace Microsoft.CodeQuality.CSharp.Analyzers.Maintainability
+{
+    /// <summary>
+    /// CA1516: Use cross-platform intrinsics
+    /// </summary>
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public sealed class CSharpUseCrossPlatformIntrinsicsAnalyzer : UseCrossPlatformIntrinsicsAnalyzer
+    {
+    }
```

Comment:
Does it need to be done this way? I think you can just apply `DiagnosticAnalyzer` attribute with both languages on the base class, and mark the base class as `sealed` instead of abstract.

Reply:
It’s intentionally only being done for C# right now due to needing some C# specific language features

VB devs aren’t really using hardware intrinsics (which often also use pointers/unsafe code) so this shouldn’t be particularly limiting

Reply:
Its split so that if someone cares enough in the future, they can go and add the relevant VB tests and support.

It's just not something I'm going to spend cycles doing as part of the initial work.